core/src/style/polylineStyle.cpp

```diff
@@ -339,6 +340,8 @@ void PolylineStyleBuilder<V>::addFeature(const Feature& _feat, const DrawRule& _
             addMesh(line, params);
         }
     } else {
+        params.closedPolygon = true;
```

Comment:
Is this necessary? I think it is implicit in the input data whether a linestring is closed (i.e. if the first point is the same as the last point, it is closed).